source/index.html

```diff
@@ -23,6 +23,14 @@
     </ul>
 
     <button id="new_comment_button">New Comment</button>
+    <div class="form">
```

Comment:
Is there a way we could have added the form using jQuery rather than HTML?

Reply:
thank you so much for the feedback 😄